documentation/modules/rn-2.6.adoc

```diff
@@ -2,6 +2,232 @@
+
+
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This part provides features and enhancements introduced in {project-full} 2.6:
+
+.Validation for RHEL 6 guest operating system is now available when migrating VMs with RHEL 6 guest operating system
+
+Red Hat Enterprise Linux (RHEL) 9 does not support RHEL 6 as a guest operating system. Therefore, RHEL 6 is not supported in {ocp-name} Virtualization. With this update, a validation of RHEL 6 guest operating system was added to {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-413[(MTV413)]
+
+.Migrations from {ocp-name} versions before 4.13 are now blocked
+
+The `Export API` function in {ocp-name} Virtualization now exposes VM manifests from {ocp-name} 4.13. Therefore, source {ocp} clusters must be version 4.13 or later. link:https://issues.redhat.com/browse/MTV-734[(MTV-734)]
+
+
+.Automatic retrieval of CA certificates for the provider's URL in the console
+
+The ability to retrieve CA certificates, which was available in previous versions, has been restortes. The `vSphere Verify certificate` option is in the `add-provider` dialog. This option was removed in the transition to {ocp} console, and has now been added to the console. This functionality is also available for RHV, OpenStack and OpenShift providers now
+link:https://issues.redhat.com/browse/MTV-737[(MTV-737)]
+
+
+.{project-short} UI plugin now uses only a dynamic plugin for the SDK REST API `fetch` methods
+
+The UI plugin now uses a dynamic plugin for the SDK REST API when using the `fetch` methods. link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
+
+
+
+.Plans that failed to migrate VMs can now be edited
+
+With this update, you can edit plans that have failed to migrate any VMs. Some plans fail or are canceled because of incorrect network and storage mappings. You can now edit these plans until they succeed. link:https://issues.redhat.com/browse/MTV-779[(MTV-779)]
+
+
+[id="ova-provider-26_{context}"]
+=== Open Virtual Appliance Provider
+
+The following are new features for the {project-short} Open Virtual Appliance (OVA) provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+
+.OVA inventory watcher detects deleted files
+
+The OVA inventory watcher detects changes in files, including deleted files. Updates from the `ova-provider-server` pod are now sent every five minutes to the `forklift-controller` pod that updates the inventory. link:https://issues.redhat.com/browse/MTV-733[(MTV-733)]
+
+.Migration from OVA validation rules
+
+VM migrations from OVA are validated according to vSphere. The migration can use a similar structure and inventory variable names as vSphere, of which the OVA model is a subset. The rules do not apply to clusters, hosts, the running state of the VMs, or any configuration that is not relevant for the OVA VMs. link:https://issues.redhat.com/browse/MTV-669[(MTV-669)]
```

Comment:
.Validation rules are now available for OVA

The validation service includes default validation rules for virtual machines from the Open Virtual Appliance (OVA).